scheduler/src/cook/mesos/api.clj

```diff
@@ -349,12 +365,21 @@
                 (mk-docker-ports docker-id port-mappings))])
       {})))
 
+(defn- build-application
```

Comment:
If you make application a component (), then you can set the value of the key :job/application to the following map

```clojure
{:application/name name
 :application/version version}
```